tests/app/test_models.py

```diff
@@ -0,0 +1,48 @@
+from app.models import EMAIL_TYPE, SMS_TYPE
+import pytest
+
+
+class TestTemplateBase:
+    @pytest.mark.parametrize(
+        'feature_enabled',
+        [
+            False,
+            True,
+        ],
+    )
+    def test_html_property_returns_none_when_feature_flag_disabled(self, mocker, sample_template, feature_enabled):
+        """
+        Test that TemplateBase.html property returns None when
+        STORE_TEMPLATE_CONTENT feature flag is disabled
+        """
+        mocker.patch('app.models.is_feature_enabled', return_value=feature_enabled)
+        mocker.patch('app.dao.templates_dao.is_feature_enabled', return_value=feature_enabled)
+        template = sample_template(template_type=EMAIL_TYPE)
+
+        if not feature_enabled:
+            assert template.html is None
+        else:
+            template.content_as_html = '<p>Some HTML content</p>'
```

Comment:
If your goal is to test the feature flag, I think this statement needs to precede the if/else block.  Does template.html return None even when template.content_as_html is set?